src/yunohost/app.py

```diff
@@ -1933,8 +1933,14 @@ def _install_appslist_fetch_cron():
 
     logger.debug("Installing appslist fetch cron job")
 
+    cron_job = []
+    cron_job.append("#!/bin/bash")
+    # We add a random delay between 0 and 60 min to avoid every instance fetching
+    # the appslist at the same time every night
+    cron_job.append("(sleep $((RANDOM%3600)); yunohost app fetchlist > /dev/null 2>&1) &")
+
     with open(cron_job_file, "w") as f:
-        f.write('#!/bin/bash\n\nyunohost app fetchlist > /dev/null 2>&1\n')
+        f.write('\n'.join(cron_job))
```

Comment:
Uh, not sure I get the question ? `char.join(list)` concatenate the element of the list with char in between. So if you have `'\n'.join(["abc", "dev"])`, you get `"abc\ndef"`